src/main/resources/application-local.yml

```diff
@@ -3,3 +3,10 @@ spring:
     url: jdbc:postgresql://localhost:5435/marketplace
     username: postgres
     password: password
+  jpa:
```

Comment:
delete all of these. (Or just not commit this file, and use it locally. But I don't recommend this way)))

In this project we don`t allow hibernate to generate tables for us. We create them  ourself using flyway migrations. 

Please learn what is Flyway and why it is used.